test/onnx/test_fx_dynamic_with_onnxruntime.py

```diff
@@ -0,0 +1,384 @@
+# Owner(s): ["module: onnx"]
+from __future__ import annotations
+
+import copy
+
+import inspect
+
+import io
+import unittest
+import warnings
+from typing import Any, Callable, Mapping, Optional, Sequence, Tuple, Union
+
+import numpy as np
+
+import onnx.reference
+import onnx_test_common
+
+import onnxruntime  # type: ignore[import]
+
+import torch
+import torchvision
+from torch.onnx._internal import _beartype, diagnostics, fx as fx_onnx
+from torch.testing._internal import common_utils
+from torch.types import Number
+from torch.utils import _pytree as pytree
+
+_NumericType = Union[Number, torch.Tensor, np.ndarray]
+_ModelType = Union[torch.nn.Module, Callable]
+_ONNXModelType = Union["onnx.ModelProto", bytes, str, io.BytesIO]
+_InputArgsType = Union[torch.Tensor, Tuple[Any, ...]]
+_InputKwargsType = Mapping[str, Any]
+_OutputsType = Union[Sequence[_NumericType], Sequence]
+
+
+@_beartype.beartype
+# TODO(titaiwang): bound.args makes pytorch_inputs hard to annotate
+# maybe annotate it when the exporter API is launched
+def _run_ort(onnx_model: _ONNXModelType, pytorch_inputs: Any) -> _OutputsType:
+    session = onnxruntime.InferenceSession(
+        onnx_model, providers=["CPUExecutionProvider"]
+    )
+    input_names = [ort_input.name for ort_input in session.get_inputs()]
+    return session.run(
+        None, {k: v.cpu().numpy() for k, v in zip(input_names, pytorch_inputs)}
+    )
+
+
+@_beartype.beartype
+def _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
```

Comment:
Missing document. I don't know what are `additional_test_inputs` and `additional_test_kwargs` for.

Reply:
Done